WPMedia/fusion-news-theme-blocks
# TMEDIA-707 : Update content sources for image resizer

## Description

Updating the:
* `collections-content-source-block`
* `content-api-source-block`
* `related-content-content-source-block`
* `search-content-source-block`
* `unpublished-content-source-block`
blocks to use the new image resizer methods.

## Jira Ticket

- [TMEDIA-707](https://arcpublishing.atlassian.net/browse/TMEDIA-707)

## Acceptance Criteria

Update the following content sources to use fetch with partial caching:
* collections-content-source-block
* content-api-source-block
* related-content-content-source-block
* search-content-source-block
* unpublished-content-source-block

## Test Steps

1. Checkout this branch `git checkout tmedia-707`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/collections-content-source-block,@wpmedia/content-api-source-block,@wpmedia/related-content-content-source-block,@wpmedia/search-content-source-block,@wpmedia/unpublished-content-source-block,@wpmedia/signing-service-content-source-block`
3. Test `3EO65DTWPNA7HHTIC3QQZB2QAM` using http://localhost/pagebuilder/tools/debugger?_website=themesinternal&type=content-debugger&source=content-api&config={%22_id%22:%223EO65DTWPNA7HHTIC3QQZB2QAM%22}
4. Check for auth tokens in return data on images.

## Effect Of Changes

### Before

### After

## Dependencies or Side Effects

https://github.com/WPMedia/arc-themes-feature-pack/pull/357

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
